concepts/basics2/about.md

```diff
@@ -0,0 +1,105 @@
+# Haskell Syntax Basics 2
+
+## Exporting functions
+
+Exporting functions was covered in the first concept, here is a quick refresher.
```

Comment:
I would argue against having these "refresher" parts of the documentation, as the concept docs should really be standalone documents. If you do feel it is needed to link to a previous concept, I'd suggest using a [concept widget](https://exercism.org/docs/building/markdown/widgets#h-concept-widget) for that.

This case is slightly special I think, where re-hashing the existing information from the basics document makes sense (but without this line). In general, you should consider the concept documents to:

- Focus on just one concept
- Only contain information on that document. Linking to other concepts can be done via concept widgets

An exercise can still unlock multiple concepts, so you're not limited to just one concept in a concept exercise.

Reply:
Understood. I agree it is better to cover a single concept and will use widgets to link to 'refreshers'. In this case however, since the concepts are so basic and universal, I followed the example of the Elm track introducing them together to make a more interesting module and exercise. 
I will remove that first line, add a widget and limit the discussion to the bare minimum, if any, on function export.